src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
-import App from "./App";
+import App from "./App.js";
```

Comment:
did you add this yourself? 🤔 

i think it should work without the `.js` since `create-react-app` has some libraries under the hood to resolve the file names

Reply:
when I add "type": "module", in package.json, it forces me to type .js in the import file names, if not it will throw an error when compiling.

Reply:
you can exclude `type: module` for create-react-app